pom.xml

```diff
@@ -65,7 +65,7 @@
 			<artifactId>jjwt-api</artifactId>
 			<version>0.11.5</version>
 		</dependency>
-        <dependency>
+       <dependency>
```

Comment:
Why are all the commits made at the same time? 
Were the commits

Reply:
Yes, Its not good to made commit at same time, But first I created whole project then one by one I made commit.